docs/changes/io.votable/17999.feature.rst

```diff
@@ -0,0 +1,2 @@
+``CooSys`` VOTable elements now have a method ``to_astropy_frame_and_time`` that returns
+the corresponding astropy built-in frame and time, when possible.
```

Comment:
```suggestion
``CooSys`` VOTable elements now have a method ``to_astropy_frame`` that returns
the corresponding astropy built-in frame when possible.
```

Reply:
OMG good catch, it's from an older version when I was returning a frame and an `astropy.time.Time` for the epoch. But the API was weird, and getting the epoch is straightforward.